### PR TITLE
vincentlaucsb-csv-parser: add version 2.2.2

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **vincentlaucsb-csv-parser/2.2.2**

https://github.com/vincentlaucsb/csv-parser/compare/2.2.0...2.2.2

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/vincentlaucsb-csv-parser/all/conanfile.py [hunk 1]
```diff
@@ -5,33 +5,52 @@
...
 
 class VincentlaucsbCsvParserConan(ConanFile):
     name = "vincentlaucsb-csv-parser"
     description = "Vince's CSV Parser with simple and intuitive syntax"
-    topics = ("conan", "csv-parser", "csv", "rfc 4180", "parser", "generator")
+    license = "MIT"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/vincentlaucsb/csv-parser"
-    license = "MIT"
+    topics = ("conan", "csv-parser", "csv", "rfc 4180", "parser", "generator")
```
Comment (RubenRBS) [inline recipes/vincentlaucsb-csv-parser/all/conanfile.py:19]:
```suggestion
    topics = ("csv", "rfc 4180", "parser", "generator")
```

**Review by RubenRBS — APPROVED** [review]
Thanks! I've edited the topics to follow the modern recommendations, thwe rest looks good, thanks for improving the recipe :)

File: recipes/vincentlaucsb-csv-parser/all/conanfile.py [hunk 2]
```diff
@@ -5,33 +5,52 @@
...
+        }
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def package_id(self):
         self.info.clear()
 
+    def validate(self):
+        # C++17 recommended: https://github.com/vincentlaucsb/csv-parser/blob/2.1.3/README.md
```
Comment (uilianries) [inline recipes/vincentlaucsb-csv-parser/all/conanfile.py:45]: Thank you for checking it!

**Review by uilianries — APPROVED** [review]
LGTM

**Pull request merged**